server/database/queries/users/addUserInterests.js

```diff
@@ -0,0 +1,14 @@
+const dbConnection = require('../../config/dbConnection');
+
```

Comment:
here the query doesn't work as expected
in the next time be sure that the data inserted as we need  
the solution to it it's arranging your variables